fetch_resources.sh

```diff
@@ -28,7 +28,11 @@ sed -i "/PYTHON_CMD=/c PYTHON_CMD=\"$TEMPEST_VENV_DIR/bin/python$PYTHON_VERSION\
 
 if [[ "$AUTH_URL" =~ "https" ]]
 then
-    OPENSTACK_CMD="openstack --insecure"
+    cd /root
```

Comment:
Do we need provision for "--insecure" flag too?

Reply:
That is automatically handled in keystone pod.